samples/README.md

```diff
@@ -1,6 +1,7 @@
 # Sample apps for the AWS IoT Device SDK for C++ v2
 
 * [Basic MQTT Pub-Sub](#basic-mqtt-pub-sub)
+* [Websocket MQTT Pub-Sub](#websocket-mqtt-pub-sub)
```

Comment:
I vote that we don't even bother with publish/subscribe in the connection samples, just do: connect, disconnect, exit.

There's quite a bit of code devoted to subscribing, sending and receiving an exact number of messages, letting users customize the path & message, etc

We can have make the pub-sub sample its own thing like the jobs and shadow sample and just have it use the default connection setup